packages/rs-dapi/src/services/platform_service/get_status.rs

```diff
@@ -303,18 +303,7 @@ fn build_chain_info(
         let core_chain_locked_height = drive_status
             .chain
             .as_ref()
-            .and_then(|c| c.core_chain_locked_height)
-            .map(|h| {
-                h.try_into()
-                    .inspect_err(|error| {
-                        tracing::warn!(
```

Comment:
why logging removed here?

Reply:
because we are going from a u32 to a u32, it can't error, so what is there to inspect?